Templates/Swift/Includes/Model.stencil

```diff
@@ -71,12 +71,12 @@ public {{ options.modelType }} {{ type }}: {% if parent %}{{ parent.type }}{% el
     {% filter indent:4 %}{% include "Includes/Model.stencil" schema %}{% endfilter %}
     {% endfor %}
 
-    public init({% for property in allProperties %}{{ property.name }}: {{ property.optionalType }}{% ifnot property.required %} = nil{% endif %}{% ifnot forloop.last %}, {% endif %}{% endfor %}) {
+    public {% if parent %}{% if properties.count == 0 %}override {% endif %}{% endif %}init({% for property in allProperties %}{{ property.name }}: {{ property.optionalType }}{% ifnot property.required %} = nil{% endif %}{% ifnot forloop.last %}, {% endif %}{% endfor %}) {
```

Comment:
In Stencil you should be able to do just `{% if parent and properties %}override {% endif %}` (If an item is an array the if checks if it exists and is not empty)
Also could you add a new model to the test fixture that has a parent but none of it's own properties

Reply:
ok but with your proposition we check if there are properties and parent, but not if we have not properties and parent to display override. right ?

Reply:
Yes you're right, I meant `{% if parent and not properties %}override {% endif %}`
Merging this now to do a release